src/components/hamburger/Sidebar.tsx

```diff
@@ -1,21 +1,36 @@
+import React, { useState } from 'react';
 import { AnimatePresence, motion } from 'framer-motion';
 
 import fbIcon from '../../assets/icons/hamburger/facebook-logo.svg';
 import igIcon from '../../assets/icons/hamburger/instagram-logo.svg';
 
+import './styles/index.css';
```

Comment:
no need to import this, the index.css is imported in the parent root component index.tsx or App.tsx